src/mockServiceWorker.js

```diff
@@ -147,20 +147,22 @@ async function handleRequest(event, requestId) {
   if (client && activeClientIds.has(client.id)) {
     ;(async function () {
       const clonedResponse = response.clone()
-      sendToClient(client, {
+      const body =
+        clonedResponse.body == null ? null : await clonedResponse.arrayBuffer()
```

Comment:
Something we may consider for future improvement is reading request/response body as streams. This way we don't have to wait for the entire body to be read in order to be sent from the worker, we can start sending as soon as we receive the first byte. 

This will require some larger refactor but may be worth it.